cmd/create/ocmrole/cmd.go

```diff
@@ -305,49 +352,69 @@ func createRoles(reporter *rprtr.Object, awsClient aws.Client, prefix string, po
 	}
 	if exists {
 		reporter.Warnf("Role '%s' already exists", roleName)
-		return roleARN, nil
-	}
-	reporter.Debugf("Creating role '%s'", roleName)
-	roleARN, err = awsClient.EnsureRole(roleName, string(policy), permissionsBoundary,
-		"", map[string]string{
-			tags.RolePrefix:  prefix,
-			tags.RoleType:    aws.OCMRole,
-			tags.Environment: env,
-		})
-	if err != nil {
-		return "", err
-	}
-	reporter.Infof("Created role '%s' with ARN '%s'", roleName, roleARN)
+		if !isAdmin {
+			return roleARN, nil
+		}
 
-	filename = fmt.Sprintf("sts_%s_permission_policy.json", aws.OCMRolePolicyFile)
-	path = fmt.Sprintf("templates/policies/%s", filename)
+		isExistingRoleAdmin, err := awsClient.IsAdminRole(roleName)
+		if err != nil {
+			return "", err
+		}
 
-	policy, err = aws.ReadPolicyDocument(path)
-	if err != nil {
-		return "", err
+		if isExistingRoleAdmin {
```

Comment:
@zgalor  I verified and works well. One thing is if the user already has admin role and they choose "No" in the response we are not warning the user that we are not making any changes to the policy. Either we can error out stating they cannot choose "No" to admin policy if it already exists or maybe a warning here that says "Admin already exists and not changes made"

Reply:
This is a good idea - if the command is idempotent, it should either set the user requirements or error out:
- `rosa create ocm-role --admin` always ensures that the ocm role is admin
- `rosa create ocm-role` always ensures that the ocm role is non-admin <--- since this is destructive if the role is admin, I would just error out and tell the user to delete the policy instead

Reply:
If it already has admin role, users will not be prompted. 
they will just get the warning `W: Role 'prefix-OCM-Role-id' already exists`, like we currently print for creating a role that already exists

Reply:
@vkareh ok. I'll go with the second option of erroring. I don't think we should revoke admin policies

Reply:
```
$ ./rosa create ocm-role --prefix zg2 --mode auto
I: Creating ocm role
I: Creating role using 'arn:aws:iam::765374464689:user/zgalor'
? Create the 'zg2-OCM-Role-12541229' role? Yes
W: Role 'zg2-OCM-Role-12541229' already exists
E: There was an error creating the ocm role: The existing role is an admin role. To remove admin capabilities please delete the admin policy and the 'rosa_admin_role' tag
```